portfolio/src/main/webapp/package-lock.json

```diff
@@ -1,13 +1,2666 @@
 {
-  "name": "webapp",
+  "name": "irisliuportfolio",
   "version": "1.0.0",
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "accepts": {
```

Comment:
I'm a little suspicious of the huge change in this file given that package.json didn't change.